pkg/types/field_name.go

```diff
@@ -43,10 +43,11 @@ const emptyName = "EMPTY_NAME"
 
 // String implements Stringer interface.
 func (name *FieldName) String() string {
-	builder := strings.Builder{}
```

Comment:
builder.Grow() should also be enough.

Reply:
I second  you, the Grow() call with precompute length of string is good enough.

Reply:
Sure, But the known problem is that stringer.builder does not have a good memory reuse mechanism. Moreover, there are already many similar slicegrows here, which need to be optimized specifically.